packages/cli/src/util/telemetry/commands/dev/index.ts

```diff
@@ -28,21 +28,21 @@ export class DevTelemetryClient extends TelemetryClient {
     }
   }
 
-  trackCliFlagYes(yes?: boolean) {
+  trackCliFlagYes(yes: boolean | undefined) {
     if (yes) {
       this.trackCliFlag('yes');
     }
   }
 
-  trackCliFlagConfirm(confirm?: boolean) {
+  trackCliFlagConfirm(confirm: boolean | undefined) {
     if (confirm) {
       this.trackCliFlag('confirm');
     }
   }
 
-  trackCliFlagHelp(help?: boolean) {
+  trackCliFlagHelp(help: boolean | undefined) {
     if (help) {
-      this.trackCliFlagHelp();
+      this.trackCliFlag('help');
```

Comment:
This audit did actually catch one bug!

Reply:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Can we have a test that asserts this? Or is it too hard to do with `vc dev` for now?

Reply:
I think this was actually me overachieving sine we planned to do `--help` as its own work. But, yeah, this is annoying to test. I could just remove it?